lib/mobile_app_backend_web/controllers/deep_link_controller.ex

```diff
@@ -1,12 +1,29 @@
 defmodule MobileAppBackendWeb.DeepLinkController do
   use MobileAppBackendWeb, :controller
 
+  defp t_alert_cta_campaign_params(),
+    do: %{
```

Comment:
I only prefer `def foo, do: bar` over `def foo do bar end` when the former actually fits on one line; in cases like this I think a `do ... end` block is cleaner than the explicit keyword list `, do:`.